### DISCUSSION TITLE
A LEVEL 13 WIKI!!!

### DISCUSSION THREAD

**Discussion by ACorridorWanderer** [nroutasuo/level13#223] [Ideas]
Alright, so I have been sharing the good news of level 13, but people don't know what to do. So I searched up a wiki but there was none. So I suggest we make a wiki! (But it might have to be named a fandom bc I made a wiki on it but didn't know how to start.)

**Comment by nroutasuo** [comment]
Nice initiative! Do you have a link for the new Wiki?

Feel free to use whatever platform that works for you, but just so you know / in case it's useful, these resources already exist that I know of:
* an extensive [FAQ on Reddit](https://www.reddit.com/r/level13/comments/p9bk89/faq_for_level_13/)
* a few Wiki pages [here on Github](https://github.com/nroutasuo/level13/wiki)

In addition a few people on the [Discord server](https://discord.gg/wz3UaRK7xZ) tried to start an external wiki awhile ago. I think that project died but you might find some people who are interested in contributing there.

**Reply by ACorridorWanderer** [reply]
Unfortunatly, under certain circumstances, I might not be able to work on it, I just wanted to present the idea (again). I am in school and stuff so that's taking my time. Our stuff is blocked. =/. (Don't worry, I'm managing my time)

**Reply by Zarioo12** [reply]
Maybe when your at home and have some spare time

**Reply by CorridorWanterer** [reply]
Yeah. Also, this is just me but I lost my previous account "ACorridorWanderer." It was such a nice name too... oh well.
